tests/cicd/jenkins_model_adapter_basic_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# init conda env
+eval "$('/opt/intel/oneapi/intelpython/latest/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
+# activate conda
+conda activate pytorch-1.12.0
+# run main.py
+cd /home/vmagent/app/e2eAIOK/e2eAIOK/ModelAdapter/src
```

Comment:
@zigzagcai , is using 'e2eAIOK' correct? I checked with my docker env, should be '/home/vmagent/app/e2eaiok/e2eAIOK'

Reply:
Updated